corehq/apps/accounting/models.py

```diff
@@ -1580,6 +1588,13 @@ def new_domain_subscription(cls, account, domain, plan_version,
                                 date_start=None, date_end=None, note=None,
                                 web_user=None, adjustment_method=None, internal_change=False,
                                 **kwargs):
+        if plan_version.subscription_set.count() >= plan_version.plan.max_domains:
```

Comment:
I might be missing some context, but do you also need to check that this limit is not reached when new domains are created? or is that what this is doing here?

Reply:
I'm pretty sure that when a new domain is created, it defaults to a CommCare Community Plan, which should never have a domain limit (@nickpell can answer more accurately though)

Reply:
@gbova is right.  (The other case is when an Advanced Trial is created for the web user's first domain, but that also won't have a max domain limit)

Reply:
got it, thanks